src/modules/colorPicker/ColorPickerUI/Helpers/ColorRepresentationHelper.cs

```diff
@@ -191,11 +193,46 @@ private static string ColorToNCol(Color color)
         /// <summary>
         /// Return a <see cref="string"/> representation of a RGB color
         /// </summary>
-        /// <param name="color">The see cref="Color"/> for the RGB color presentation</param>
+        /// <param name="color">The <see cref="Color"/> for the RGB color presentation</param>
         /// <returns>A <see cref="string"/> representation of a RGB color</returns>
         private static string ColorToRGB(Color color)
             => $"rgb({color.R.ToString(CultureInfo.InvariantCulture)}"
              + $", {color.G.ToString(CultureInfo.InvariantCulture)}"
              + $", {color.B.ToString(CultureInfo.InvariantCulture)})";
+
+        /// <summary>
+        /// Returns a <see cref="string"/> representation of a CIE LAB color
+        /// </summary>
+        /// <param name="color">The <see cref="Color"/> for the CIE LAB color presentation</param>
+        /// <returns>A <see cref="string"/> representation of a CIE LAB color</returns>
+        private static string ColorToCIELAB(Color color)
+        {
+            var (lightness, chromaticityA, chromaticityB) = ColorHelper.ConvertToCIELABColor(color);
+            lightness = Math.Round(lightness);
+            chromaticityA = Math.Round(chromaticityA);
+            chromaticityB = Math.Round(chromaticityB);
+
+            return $"L*a*b*({lightness.ToString(CultureInfo.InvariantCulture)}" +
+                   $", {chromaticityA.ToString(CultureInfo.InvariantCulture)}" +
+                   $", {chromaticityB.ToString(CultureInfo.InvariantCulture)})";
```

Comment:
Is this a standard format? I've seen the following format online `CIELab(97.00, 2.25, 5.73)`

Reply:
CSS 4 specifies the format `lab(<percentage> <number> <number>)` (e.g. `lab(62.2345% -34.9638 47.7721)`)